lib/core/index.js

```diff
@@ -25,7 +26,8 @@ function dashesCamelCase (str: string) {
 }
 
 export const getFilePath = (context, styleFilePath) => {
-  const settings = context.settings['css-modules'];
+
+  const settings = context.settings && context.settings['css-modules'];
```

Comment:
unrelated to this PR but just makes it fail-safe if settings doesn't exist in .eslintrc config